dataland-frontend/tests/e2e/specs/search-company-framework-data/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -103,39 +104,39 @@ describe("As a user, I expect the search functionality on the /companies page to
     "Checks that the sector filter synchronises between the search bar and the drop down and works",
     { scrollBehavior: false },
     () => {
-      const demoCompanyToTestFor = companiesWithEuTaxonomyDataForNonFinancials[0].companyInformation;
+      const demoCompanyToTestFor = assertDefined(
```

Comment:
Warum hat das hier ein `assertDefined()`, jedoch `demoCompanyWithDifferentSector` nicht?

Wenn beide übrigens ein `assertDefined` haben, brauchen wir die zwei `excpect()` checks unten ja doch nicht oder?

Reply:
Ich hab es jetzt ohne `assertDefined`s gelöst und die defined checks mit in die `expect`s eingebaut ist. Ohne `!`s bin ich nicht ausgekommen, weil die IDE sonst nicht happy ist. To be pushed after linting.

Reply:
Nope, der linter mag hier `!`s nicht. `assertDefined` it is.